src/ensembl/production/metadata/api/genome.py

```diff
@@ -80,9 +80,46 @@ def fetch_taxonomy_ids(self, taxonomy_names):
                 taxids.append(taxid[0])
         return taxids
 
-    def fetch_genomes(self, genome_id=None, genome_uuid=None, assembly_accession=None, ensembl_name=None,
-                      taxonomy_id=None, group=None, group_type=None, unreleased_only=False, site_name=None,
-                      release_type=None, release_version=None, current_only=True):
+    def fetch_genomes(self, genome_id=None, genome_uuid=None, assembly_accession=None, assembly_name=None,
+                      ensembl_name=None, genebuild_id=None, taxonomy_id=None, group=None, group_type=None,
```

Comment:
genebuild_id is not implemented and should not be part of this PR.
This could be it's own method within the class as it isn't trivial to get. It will also only ever return one genome so it could be interesting.
Better might be dataset_name or other aspect of dataset. Those could be very useful.